scripts/run-dependencies

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+
+HERE=$(realpath "$(dirname $0)")
+
+# Build and run the database
+# TODO: Make build step skippable
+"$HERE"/../db/scripts/build
```

Comment:
As the db image won't change that much (because the schema is the bit that changes and not baked into the image) we might build this elsewhere. leave it this way for now perhaps and we can easily split it out later